dcat/rdf/dcat.ttl

```diff
@@ -844,14 +937,15 @@ dcat:mediaType
   skos:editorialNote "Status: English Definition text modified by DCAT revision team, Italian translation provided, other translations pending. Note some inconsistency on def vs. usage"@en ;
   skos:changeNote "The range of dcat:mediaType has been tightened as part of the revision of DCAT."@en ;
   skos:scopeNote "This property SHOULD be used when the media type of the distribution is defined in the IANA media types registry https://www.iana.org/assignments/media-types/, otherwise dct:format MAY be used with different values."@en ;
+  skos:scopeNote "Esta propiedad DEBERÍA usarse cuando el 'media type' de la distribución está definido en el registro IANA de 'media types' https://www.iana.org/assignments/media-types/, de lo contrario, dct:format PUEDE usarse con distintos valores."@es ;
   skos:scopeNote "Questa proprietà DEVE essere usata quando il tipo di media della distribuzione è definito nel registro dei tipi di media IANA https://www.iana.org/assignments/media-types/, altrimenti dct:format PUO 'essere usato con differenti valori"@it
 .
 dcat:compressFormat
   rdf:type rdf:Property ;
   rdf:type owl:ObjectProperty ;
   rdfs:comment "The format of the distribution in which the data is contained in a compressed form, e.g. to reduce the size of the downloadable file."@en ;
   rdfs:comment "El formato de la distribución en el que los datos están en forma comprimida, e.g. para reducir el tamaño del archivo a bajar."@es ;
```

Comment:
if the PR #986 is merged, the spanish translation should change as in the following
```suggestion
  rdfs:comment "El formato del archivo en el que los datos están en forma comprimida, e.g. para reducir el tamaño del archivo a bajar."@es ;
```

Reply:
Thank @riccardoAlbertoni - I'll check #986